src/Repository/AdherentRepository.php

```diff
@@ -590,6 +591,31 @@ public function countMembersByGenderForReferent(Adherent $referent): array
         return $this->formatCount($result);
     }
 
+    public function countCommitteeMembersInReferentManagedArea(Adherent $referent, StatisticsParametersFilter $filter = null, int $months = 5): array
+    {
+        $this->checkReferent($referent);
+
+        $query = $this->createQueryBuilder('adherent', 'adherent.gender')
+            ->select('COUNT(DISTINCT adherent.id) AS count, YEAR_MONTH(event.beginAt) as yearmonth')
+            ->join('adherent.memberships', 'membership')
+            ->join('membership.committee', 'committee')
+            ->innerJoin('committee.referentTags', 'tag')
+            ->where('tag IN (:tags)')
+            ->andWhere('committee.status = :status')
+            ->andWhere('membership.joinedAt >= :from')
+            ->andWhere('membership.joinedAt <= :until')
+            ->setParameter('tags', $referent->getManagedArea()->getTags())
+            ->setParameter('status', Committee::APPROVED)
+            ->setParameter('until', (new Chronos('now'))->setTime(23, 59, 59, 999))
+            ->setParameter('from', (new Chronos("first day of -$months months"))->setTime(0, 0, 0, 000))
+            ->groupBy('yearmonth')
```

Comment:
Let's add a little bit of cache WDYT? Like 1 hour or 30 minutes?

Reply:
Yes, I add 1 hour.